### DISCUSSION TITLE
다른 채팅방으로 메시지 보내기

### DISCUSSION THREAD

**Discussion by sgim87554** [storycraft/node-kakao#399] [Q&A]
v3에서  채팅을 다른 채팅방으로 보낼 방법이 있나요

**Comment by ghost** [comment]
네. 있습니다.

**Reply by jipsoft2** [reply]
참으로 개발자스러운 답변이군요~ ㅋㅋ
마치 올 때 우유 사와. 계란 있으면 30개 사와.
if (existEgg == true) {
  buyMilk(30);
} else {
  buyMilk(1);
}
같은 느낌이랄까요.

질문하신 분은 방법이 있는지를 질문했지만 사실상 방법이 궁금하신 것 같은데..

@cat-123-cell 보내고 싶은 채팅방의 channel ID를 기억하고 있다가 (혹은 실시간으로 찾아내도 되죠)
```
var channel = client.channelManager.get(id);
channel.sendText(chat.text);
```
같은 식으로 보내시면 됩니다.

**Comment by generalxd** [comment]
찾아보고 물어보는게 좋을듯합니다.
